2.Installation/2.3.InstallUsingComposer.md

```diff
@@ -0,0 +1,16 @@
+2.1 Installing using Composer
+=============================
+ 
+Installing Fluid Powered TYPO3 with Composer is a simple command-line operation:
+
+Prerequisites: 
+- You have to use TYPO3 in Composer-mode. Please look at https://composer.typo3.org/ for more information. 
+- It is recommended that you have added `https://composer.typo3.org/` to your list of repositories.
+
+1. Open a command-line tool and navigate to the folder where the composer.json for your TYPO3-installation exists.
+2. Execute the command: `composer require typo3-ter/fluidcontent`.
+3. Any dependencies automatically get installed.
+
+Composer resolves all dependencies of extensions that have their dependencies set correctly. For example, installing this site's Provider Extension (EXT:fluidtypo3org) will immediately also install `typo3-ter/flux`, `typo3-ter/vhs`, `typo3-ter/fluidpages` and `typo3-ter/fluidcontent`.

```

Comment:
This should read `fluidtypo3/fluidpages` etc. - in other words, we only recommend the packagist names and discourage installing from TER.